### ISSUE TITLE
cpptraj give the wrong topology original filename if using relative dir

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#329]
pytraj test breaks

```python
In [3]: top_file = '../cpptraj/test/Test_Matrix/1rrb_vac.prmtop'

In [4]: top = pt.load_topology(top_file)

In [5]: top.filename
Out[5]: '/Users/haichit/pytraj_github/pytraj/tests'
```

https://github.com/Amber-MD/pytraj/blob/master/pytraj/topology.pyx#L433-L448

**Comment by hainm** [comment]
this happens with #328

**Comment by hainm** [comment]
absolute path is fine.

**Comment by hainm** [comment]
actually I don't know what's going on. It only happens with `1rrb_vac.prmtop` file :D 

```python
In [15]: top = pt.load_topology('./1rrb_vac.prmtop')

In [16]: top.filename
Out[16]: '/Users/haichit/pytraj_github/pytraj/cpptraj/test/Test_Matrix'
```

**Comment by drroe** [comment]
Should be fixed by #330

**Comment by hainm** [comment]
thanks. confirmed.
